.dockerignore

```diff
@@ -3,7 +3,6 @@ deps/
 charts/
 .vscode/
 log/
-.git/
```

Comment:
we really shouldn't be including the git subfolder in our images

Reply:
(if we can get around tagging the version with just an env var or something similar being there at build, I think that's fine)

Reply:
the .git folder actually isn't very large and it never makes it into the final image.

Reply:
Oh that is true it's only used in the builder I suppose.  Still feel like the approach is going to bite us at some point (and github actions has a canonical env var for tag I believe)

Reply:
It hasn't caused problems with the console. An env var seems less stable than the git tags, also between environments.